trl/trainer/grpo_trainer.py

```diff
@@ -324,7 +419,6 @@ def data_collator(features):  # No data collation is needed in GRPO
                         enable_prefix_caching=True,
                     )
                 self.sampling_params = SamplingParams(
-                    n=self.num_generations,
```

Comment:
now, the prompt is repeated, so the generation will be called `G` times. I should be equivalent in term of compute thanks to `enable_prefix_caching=True,`